projects/igniteui-angular/src/lib/test-utils/template-strings.spec.ts

```diff
@@ -75,6 +75,13 @@ export class ColumnDefinitions {
         <igx-column field="HireDate" pinned="true"></igx-column>
     `;
 
+    public static idNameHiddenJobHire = `
```

Comment:
You may consider using the columns definition just above this new one and simply unpin the last column.